projects/plugins/jetpack/extensions/blocks/ai-assistant/edit.js

```diff
@@ -50,6 +64,10 @@ export const createPrompt = (
 	userPrompt = '',
 	type = 'userPrompt'
 ) => {
+	if ( ! postTitle?.length ) {
+		return '';
+	}
+
```

Comment:
The lack of title shouldn't be a blocker for generating the prompt. If there's a new post without a title set it should still work